docs_api/conf.py

```diff
@@ -112,7 +112,7 @@
 
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".
-html_title = "Low-level API for h5py"
+html_title = "Low-level API for h5py (v{})".format(release)
```

Comment:
N.B. the title is what shows up in the tab. The end of this is often hidden - I use an extension to have a list of tabs in the sidebar, but I still wouldn't see the ending of this:

![Screenshot from 2020-08-02 13-28-28](https://user-images.githubusercontent.com/327925/89123027-1d641c00-d4c4-11ea-89df-7b4414051c54.png)

It's also possible to put the version in the content as `|version|` - [Sphinx docs](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#default-substitutions) - which might be useful, at least on the front page.